content/docs/usage/setup/configuration.mdx

```diff
@@ -2,4 +2,55 @@
 title: Configuration
 ---
 
-Coming Soon.
+
+## Configuration
+
+After the installation, you'll need to go through the initial configuration,
+
+#### 1) Terms [#configuration-1]
+
+Welcome to Dogebox! Please read and accept the terms and conditions
+
+![](../../../../public/static/image/docs/configuration/configuration-terms.png)
+
+#### 2) Config [#configuration-2]
+
+Set your device name and preferred keyboard layout.
```

Comment:
Should we mention that setting the device name allows access using mDNS `device-name.local:8080`?

Maybe mention that configuring keyboard layout is needed for interacting with the system through `ssh` or with a monitor and keyboard attached